adala/skills/collection/classification.py

```diff
@@ -70,9 +110,8 @@ class ClassificationSkill(TransformSkill):
     def validate_response_model(self):
 
         if self.response_model:
-            raise NotImplementedError(
-                "Classification skill does not support custom response model yet."
-            )
+            # NOTE: the field_schemas provided are not the same as response_model.schema() (json schemas don't roundtrip), so can't validate here
```

Comment:
Maybe it's clearer to just comment out the whole validation function with this comment then?